src/components/molecules/EarthEditor/OutlinePane/hooks.tsx

```diff
@@ -384,6 +386,7 @@ export default ({
     visibilityShown: true,
     selectedLayerId,
     rootLayerId,
+    onZoomToLayer: onZoomToLayer,
```

Comment:
You only need to do something: something123 if the prop name is different from the function/variable's name

```suggestion
    onZoomToLayer,
```